__device-tests__/gutenberg-editor-writing-flow.test.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Internal dependencies
+ */
+const { blockNames } = editorPage;
+
+describe( 'Gutenberg Editor - Writing Flow', () => {
+	describe( 'TC007 - Test format detection under the cursor', () => {
```

Comment:
Do we believe there is benefit in retaining the `TC###` labels? My thought is that were useful when test cases were manual and needed to be referenced frequently by humans. However, for automated tests, I'm unsure they provide value. WDYT?

Reply:
I see both sides here, but feel that leaving the labels in does still provide some value while we are transitioning between manual and automated test cases. While the [Rich Text Formatting test cases](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md) are still extant, I feel that keeping the `TC###` numbers helps as a shorthand to refer and differentiate between the remaining seven test cases. Eventually, if we were to remove that doc altogether and only use automated tests, I think it would make sense to drop the TC### values from the test descriptions.